executor/adapter.go

```diff
@@ -444,6 +445,19 @@ func (a *ExecStmt) LogSlowQuery(txnTS uint64, succ bool) {
 	}
 }
 
+func (a *ExecStmt) getStatsInfo() map[string]uint64 {
+	if a.Plan == nil {
+		return nil
+	}
+	physicalPlan, ok := a.Plan.(plannercore.PhysicalPlan)
```

Comment:
Some other plan, for example `Insert`/`Update`, they are not `PhysicalPlan` but can have StatsInfo as well. For example, `insert into .. select ..`

Reply:
em... any advice for this?